### ISSUE TITLE
Sources stop working seemingly randomly

### ISSUE THREAD

**Issue by MhmdFVC** [bozbez/win-capture-audio#87]
Had an issue today where I closed OBS during a stream accidentally, and when I started it back up one of two of my audio sources with the plugin stopped working. A few minutes after fixing it, the other one (game audio) stopped working after a game restart.

**Comment by XenSide** [comment]
Yeah it seems that restarting the window you're recording makes the audio source not work anymore, my guess is that only the PID is used and not the window name so when you restart your game the PID changes and the plugin no longer records audio from that game.

My suggestion would be to, once no audio has been recorded for x amount of time, look for windows with the same window name (instead of PID) and see if a new PID has been assigned.
